transformers/uws/server.js

```diff
@@ -24,7 +24,7 @@ module.exports = function server() {
   }, this.primus.options.transport);
 
   const group = native.server.group.create(
-    opts.perMessageDeflate ? 1 : 0,
+    opts.perMessageDeflate,
```

Comment:
How about using `opts.perMessageDeflate >>> 0` to make sure that an integer is passed to the native layer? Or is it already done there?

Reply:
The addon always receives an integer no matter if it makes sense or not from JS land

Reply:
Sure. If that is how all other transports work then go ahead